src/frequenz/channels/_latest_value_cache.py

```diff
@@ -78,34 +134,53 @@ def unique_id(self) -> str:
         """The unique identifier of this instance."""
         return self._unique_id
 
-    def get(self) -> T_co:
+    def get(self, key: HashableT | None = None) -> T_co:
```

Comment:
So we can't use `None` as a key :thinking: 

I guess eventually we'll want it, so maybe implement this using a sentinel from the start? :D